spring-backend/src/main/kotlin/com/example/sztoswro/member/MemberController.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.sztoswro.member
+
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class MemberController(val memberService: MemberService): MemberAPI {
+    override fun getAll(): List<MemberDTO> {
```

Comment:
Nie chcemy zwracać haseł (chyba ze w konfiguracji jakos się to usuwa :D)

Reply:
Możemy ewentualnie dać osobne dto do memberów, a osobne do credentiali